draft-ietf-tls-tls13.md

```diff
@@ -2939,6 +2939,12 @@ servers MUST process the client's ClientHello and then immediately
 send the ServerHello, rather than waiting for the client's
 EndOfEarlyData message.
 
+Servers are permitted to buffer and postpone the processing of application
+data sent in 0-RTT until they receive client's Finished. Therefore, when
```

Comment:
It might help to at least hint at why this strategy might be useful.  That is, if this treatment is uniformly applied by a server, the data to which it is applied cannot ever be replayed, even though TLS does not provide replay protection.